source/scss/layout/_header.scss

```diff
@@ -3,15 +3,22 @@
 // TODO font-size and padding/margin variables should go here, speak to your designer, to find out, if we use the sizes in a more global context
 body > header {
   background-color: $white;
-  display: flex;
+  display: block;
   margin: 0 auto;
+  position: fixed;
+  top: 0;
   width: 100%;
+  z-index: 1150;
```

Comment:
Maybe it makes sense to have this z-index as a variable, so we can easily check that we have more components under the same z-index in the future and/or create new ones under the same level, like the `backdrop` class for example. What do you think?